site/src/App.js

```diff
@@ -73,6 +74,11 @@ const App = () => {
               <a href="https://slack.layer5.io/" >Join Our Open Source Community</a>
             </div>
           </section>
+          <section className="faq">
+            <h1>Frequently Asked Questions</h1>
+            <Faq category={["Meshery Playground"]}/>
+            {/* <Faq /> */}
```

Comment:
If the commented code serves no real purpose other then when you were testing locally , please remove it to avoid redundancy and keep everything clean, thanks